crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,12 +1276,20 @@ def poweron
 
   def powercycle
     set_state("reboot")
-    bmc_cmd("power cycle")
+    if @node[:platform] == "windows"
+      net_rpc_cmd(:power_cycle)
```

Comment:
The calls for `reboot` and `powercycle` should be slightly different (at least, they are for non-windows nodes: `reboot` vs `reboot -f`). Do I understand right that the `-f` parameter to `net` is also used to force a shutdown/reboot? In that case, we probably don't want to do it in `reboot`/`shutdown`, but only in `powercycle`/`poweroff`.